corehq/apps/adm/models.py

```diff
@@ -584,22 +588,21 @@ class CaseFilterADMColumnMixin(DocumentSchema):
     case_types = ListProperty()
     case_status = StringProperty(default='', choices=[s[0] for s in CASE_STATUS_OPTIONS])
 
-    def get_filtered_cases(self, domain, user_id, include_groups=True, status=None, include_docs=False, datespan_keys=None):
-        if not datespan_keys:
-            datespan_keys = [[], [{}]]
+    def get_filtered_cases(self, domain, user_id, status=None,
+                           date_range=None):
```

Comment:
@biyeun I decided to go ahead with the refactor from bullet 2, assuming this code's used. Let me know still whether you think this code is used.